CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+## v1.0.0
```

Comment:
```suggestion
## v1.0.0-alpha
```

Reply:
From the slack release thread:
**zeus** *1 hour ago* shoot I guess I should've generated the changelog with v1.0.0-alpha.1 instead of v1.0.0 :(
**asmacdo**  *1 hour ago* seems fine to me, do we need a separate section for the alpha?
**estroz**  *1 hour ago* i don't think so no

Reply:
Fixed, it is now v1.0.0-alpha.1